tests/foreman/api/test_activationkey.py

```diff
@@ -265,25 +198,13 @@ def test_update_max_content_hosts(self):
         @Assert: The update fails with an HTTP 422 return code.
 
         """
-        attrs = entities.ActivationKey(max_content_hosts=1).create_json()
-        path = entities.ActivationKey(id=attrs['id']).path()
-        new_attrs = attrs.copy()
-        new_attrs['max_content_hosts'] = 'foo'
-        response = client.put(
-            path,
-            new_attrs,
-            auth=get_server_credentials(),
-            verify=False,
-        )
-        self.assertEqual(
-            response.status_code,
-            httplib.UNPROCESSABLE_ENTITY,
-            status_code_error(path, httplib.UNPROCESSABLE_ENTITY, response),
-        )
-
-        # Status code is OK. Was `max_content_hosts` changed, or is it still 1?
-        response = entities.ActivationKey(id=attrs['id']).read_json()
-        self.assertEqual(response['max_content_hosts'], 1)
+        act_key = entities.ActivationKey(max_content_hosts=1).create()
+        with self.assertRaises(HTTPError):
+            entities.ActivationKey(
+                id=act_key.id,
+                max_content_hosts='foo',
```

Comment:
Same here.
If that statement is correct, so we don't have any value type verification that can be subject for further improvements.